backend/apps/users/models.py

```
@@ -0,0 +1,117 @@
+class DoctorProfile(models.Model):
+    user = models.OneToOneField('users.Users', on_delete=models.CASCADE)
+    specialization = models.CharField(max_length=255)
+    hospital = models.CharField(max_length=255)
+    phone = models.CharField(max_length=15)
+    address = models.TextField()
+    country = models.CharField(max_length=255, default='Bangladesh')
+    dob = models.DateField()
+    image = CloudinaryField('image', null=True, blank=True)
+    gender = models.CharField(max_length=10, choices=gender)
+    
+    def __str__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)